docs/manual-EN/3.build-develop-and-administration/3.configurations/0.system-requirement.md

```diff
@@ -60,7 +60,7 @@ Take AWS EC2 c5d.xlarge as an example:
 
 ## About HDD and Gigabit Networks
 
-**Nebula Graph** is designed for NVMe SSD and 10 Gigabit Network. There is no special adaptation for HDD and gigabit networks. The following are some parameters to be tuned:
+Nebula Graph is designed for NVMe SSD and 10 Gigabit Network. There is no special adaptation for HDD and gigabit networks. The following are some parameters to be tuned:
```

Comment:
is designed for NVMe SSD and 10 Gigabit Network. ==> is intended for NVMe SSD and 10 Gigabit Network.

I think "intended" may be more accurate?

Reply:
I think both are ok